### PR TITLE
User story 19 dk

### LINKED ISSUE #21: User Story 19, User Profile Show Page
As a registered user
When I visit my profile page
Then I see all of my profile data on the page except my password
And I see a link to edit my profile data

### PR DESCRIPTION
**Pull request by KirbyDD**
## Description
Resolves #21 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which maintains existing functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## RSpec Results
```
Finished in 4.72 seconds (files took 3.03 seconds to load)
127 examples, 0 failures

Coverage report generated for RSpec to /Users/tronkat/Turing/BackEnd/Mod2/projects/monster_shop_part_1/coverage. 1700 / 1700 LOC (100.0%) covered.
```

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,6 @@
-<h1>Arrrgghhh. Ahoy <%= @user.name %></h1>
+<section>
```
Comment (philjdelong) [inline app/views/users/show.html.erb:1]: I feel like you've done this before..

**Review by philjdelong — APPROVED** [review]
Looks good!

**Review by mel-rob — APPROVED** [review]
Looks good!

**Review by DanielEFrampton — APPROVED** [review]
Lookin' good

**Pull request merged**